modules/site-health/audit-full-page-cache/load.php

```diff
@@ -0,0 +1,198 @@
+<?php
+/**
+ * Module Name: Audit Full Page Cache
+ * Description: Adds a Full page Cache checker in Site Health checks.
+ * Experimental: No
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ */
+
+/**
+ * Require helper functions.
+ */
+require_once __DIR__ . '/helper.php';
+
+/**
+ * Adds tests to site health.
+ *
+ * @since 1.0.0
+ *
+ * @param array $tests Site Health Tests.
+ * @return array
+ */
+function perflab_afpc_add_full_page_cache_test( $tests ) {
+	$tests['async']['perflab_page_cache'] = array(
+		'label'             => esc_html__( 'Page caching', 'performance-lab' ),
```

Comment:
I believe `label` is exported to JSON and is then injected with JS into the DOM in a way that does not deal properly with HTML tags and HTML entities. Therefore, I don't believe any escaping should be included here.

```suggestion
		'label'             => __( 'Page caching', 'performance-lab' ),
```

Reply:
Or even I don't know where this label is displayed, as there is another `badge.label` below which is probably what I have in mind here.

Reply:
you are right, I double-checked how they do in the core.